.github/workflows/dev_environment.yml

```diff
@@ -80,7 +80,7 @@ jobs:
             llvm_commit=${{ steps.build_info.outputs.llvm_commit }}
             toolchain=${{ inputs.toolchain }}
           push: ${{ inputs.container_registry != '' }}
-          load: true
+          load: ${{ inputs.build_cuda_quantum || inputs.test_cuda_quantum || inputs.export_environment }}
```

Comment:
Unfortuantely, the action to build and push a docker image does not allow to both load and push at this time.